src/utils.py

```diff
@@ -8,10 +8,11 @@
 import logging
 import time
 from operator import itemgetter
+from os import getpid
 
 from . import read_config, zk
 from .exceptions import SwitchoverException, FailoverException
-from .helpers import app_name_from_fqdn
+from .helpers import app_name_from_fqdn, get_hostname
```

Comment:
NIT: after importing `get_hostname`, it is used in both ways: `get_hostname()` and `helpers.get_hostname()`, not qute cute :(

Reply:
> NIT: after importing `get_hostname`, it is used in both ways: `get_hostname()` and `helpers.get_hostname()`, not qute cute :(

No, it's not one big file, it's different file, so it is irrelevant.